### PR TITLE
Include all the points into consistency check for 3d cluster

### PR DESCRIPTION
**Pull request by tellet-q**
Let this run until an updated version of the test is ready

* Include all 200K into consideration
* Skip vector, only check for payload

### CODE REVIEW

**Review by timvisee — APPROVED** [review]
Note that this is using the existing consistency check script, but for 200k points.

The new consistency check script being more resilient to ongoing changes will be worked on in a separate approach.

File: tools/local/check-cluster-health.sh [hunk 1]
```diff
@@ -35,10 +35,18 @@ search_operational=$([ $exit_code -eq 0 ] && echo true || echo false)
...
 
-for _ in {1..5}; do
-  tools/check-consistency.py &
-  pids+=($!)
-done
+if [ "$QC_NAME" == "qdrant-chaos-testing-three" ]; then
+  for _ in {1..5}; do
+    tools/check-consistency-all-points.py &
+    pids+=($!)
+  done
```
[the remark this answers is not in the document]
Comment (tellet-q) [reply tools/local/check-cluster-health.sh:42]: To check 500 points instead of just 100 per run. As it's triggering 5 consistency checks in parallel.

**Comment by tellet-q** [comment]
> Note that this is using the existing consistency check script, but for 200k points. We just enable it on the 3rd cluster to see what happens.
> 
> The new consistency check script being more resilient to ongoing changes will be worked on in a separate approach.

Yes, I added a new approach as a separate module: `check-consistency-all-points-all-peers.py` but haven't enabled it in Chaos Testing

File: tools/local/check-cluster-health.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (KShivendu) [reply tools/local/check-cluster-health.sh:42]: I haven't checked the code but judging by the name `check-consistency-all-points.py` I believe it's checking all 200K points in one go. So these 5 parallel attempts shouldn't be required for this one.

**Pull request merged**